api/observability/v1/output_types.go

```diff
@@ -769,3 +774,50 @@ type Syslog struct {
 	// +kubebuilder:validation:Optional
 	MsgID string `json:"msgID,omitempty"`
 }
+
+type OtlpTuningSpec struct {
+	BaseOutputTuningSpec `json:",inline"`
+
+	// Compression causes data to be compressed before sending over the network.
+	// It is an error if the compression type is not supported by the output.
+	//
+	// +kubebuilder:validation:Enum:=gzip;none
+	// +kubebuilder:default:=gzip
+	Compression string `json:"compression,omitempty"`
+}
+
+// Otlp provided configuration for sending OTLP/HTTP with JSON-encoded Protobuf payload requests.
```

Comment:
The resourceLog format we send is protobuf mapped to JSON.   https://opentelemetry.io/docs/specs/otlp/#otlphttp
"OTLP/HTTP uses Protobuf payloads encoded either in binary formator in JSON. Regardless of the encoding the Protobuf schema of the messages is the same for OTLP/HTTP and OTLP/gRPC"

Reply:
If you think we should remove the word 'protobuf' due to confusion, I could agree with that.

Reply:
The spec defines the encoding.  I would prefer to remove any mention of it here because we are not using it.  The payload may be acceptable for it, we can integrate protobuf...but dont.  I think it makes more sense to remove the notion and add the comment in later if we actually do use it